ui-load-tester.rb

```diff
@@ -0,0 +1,29 @@
+# What is this:
```

Comment:
suggestion: I wonder if we should find a better location for this script. I'm cool with it living in this repo, but could we create a directory for things like scripts? This being the first item there